stacked-predictions/readme.md

```diff
@@ -0,0 +1,20 @@
+# Stacked Predictions
+
+This script takes two datasets, a training and a holdout, and builds
+several predictors using the training dataset. These include a model,
+a bagging ensemble, a random decision forest, a boosted tree, and a
```

Comment:
Heh, I got used to using "boosted trees" when writing all those blog posts for the release. It makes less sense to me, but was the branding I was asked for.